src/components/Tab/TabButton.tsx

```diff
@@ -0,0 +1,27 @@
+import { ReactNode } from "react";
+import { tabButtonStyle } from '@/components/Tab/index.css';
+
+interface TabButtonProps {
+  children: ReactNode;
+  isSelected: boolean;
+  onClick: () => void;
+  colorScheme?: 'main' | 'dark';

```

Comment:
'main' | 'dark'는 직관적이지 못한 느낌이 드는데 다음과 같이 수정하는건 어떨까요?
```suggestion
  colorScheme?: 'primary' | 'secondary';
```